cmd/wallet-web/src/pages/CredentialDetails.vue

```diff
@@ -104,10 +107,44 @@ export default {
       toggleDelete,
     };
   },
-  computed: {
-    ...mapGetters(['getProcessedCredentialByID']),
-    credential() {
-      return this.getProcessedCredentialByID(this.$route.params.id);
+  data() {
+    return {
+      loading: true,
```

Comment:
Where is this loading getting used in this file ?

Reply:
We need to add the loading state in the UI which this variable will be used for as a condition

Reply:
Filed https://github.com/trustbloc/wallet/issues/1295